### PR TITLE
Create t-completion for sferik/t CLI

### PR DESCRIPTION
**Pull request by chrisopedia**

### CODE REVIEW

File: t-completion.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+class TCompletion < Formula
+  homepage "http://sferik.github.com/t"
```
Comment (bfontaine) [inline t-completion.rb]: Could you please add a short description above this line using `desc "…"`? Thanks!

File: t-completion.rb [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+  url "https://github.com/sferik/t/archive/v2.9.0.tar.gz"
+  sha256 "310da36e655b9bb41032a206dc7301eb2c38cc5e0f57f7a2b2dea03d4c2fb30c"
+  head "https://github.com/sferik/t.git"
+
+  def install
+    bash_completion.install "etc/t-completion.sh" => "t"
+    zsh_completion.install "etc/t-completion.zsh" => "_t"
+  end
```
Comment (bfontaine) [inline t-completion.rb]: Please add a test. The following should probably work:

```
  test do
    assert_match "-F _t",
      shell_output("source #{bash_completion}/t && complete -p t")
  end
```

File: t-completion.rb (hunk 1 above)
Comment (chrisopedia) [inline t-completion.rb]: Done.

File: t-completion.rb (hunk 2 above)
Comment (chrisopedia) [inline t-completion.rb]: Done.

File: t-completion.rb [hunk 3]
```diff
@@ -0,0 +1,17 @@
+class TCompletion < Formula
+  desc "A command-line power tool for Twitter."
```
Comment (bfontaine) [inline t-completion.rb]: Please describe the formula, not the tool. The formula installs the completion scripts for `t`; not `t` itself.

**Comment by bfontaine** [comment]
@BrewTestBot test this please.

**Comment by bfontaine** [comment]
@BrewTestBot test this please.

File: t-completion.rb [hunk 4]
```diff
@@ -0,0 +1,17 @@
+class TCompletion < Formula
+  desc "Installs completion scripts for command-line power tool for Twitter."
```
Comment (bfontaine) [inline t-completion.rb]: Please shorten the description. “`homebrew/completions/t-completion: <desc>`” should fit under 80 characters.

**Comment by bfontaine** [comment]
Thank you for your contribution to Homebrew @chrisopedia; we appreciate it! :smile:

**Pull request closed without merging**